### ISSUE TITLE
weight according to initial loss scale

### ISSUE THREAD

**Issue by smilesun** [marrlab/DomainLab#862]
for each regularization loss, before training start, calculate the ratio of this loss term with respect to the task loss (cross entropy for classiifcaiton) 

e.g. regularization loss for DANN is https://github.com/marrlab/DomainLab/blob/1df808a151b72a198344df8f0ffec7a4b2f71222/domainlab/models/model_dann.py#L108.

However, we do not need to care about this line above, but rather, 
https://github.com/marrlab/DomainLab/blob/1df808a151b72a198344df8f0ffec7a4b2f71222/domainlab/models/a_model.py#L52

This issue is to make a feature for DomainLab such that for each loss term, we have a reference ratio of this loss term over the task loss, then the user input gamma_reg will be interpreted as a multiplication factor only on top of this ratio,  i.e. 
list_multiplier[j]*ratio[j], where j is the index for the loss term in question, ratio is something we have to calculate before training starts, here: https://github.com/marrlab/DomainLab/blob/1df808a151b72a198344df8f0ffec7a4b2f71222/domainlab/algos/trainers/train_basic.py#L22

we probalby need to define something like self.list_reg_loss_over_task_loss_ratio in https://github.com/marrlab/DomainLab/blob/1df808a151b72a198344df8f0ffec7a4b2f71222/domainlab/models/a_model.py#L17

then before https://github.com/marrlab/DomainLab/blob/1df808a151b72a198344df8f0ffec7a4b2f71222/domainlab/models/a_model.py#L52

we do new_list_multiplier= [mtuple[0]*mtuple[1] for mtuple in zip(self.list_reg_loss_over_task_loss_ratio, list_multiplier)]

**Comment by smilesun** [comment]
working on a pr https://github.com/marrlab/DomainLab/pull/863
